build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.targets

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <Target Name="BinPlaceBootstrapDll" Condition="'$(AppxPackage)' != 'true' and '$(OutputType)' != 'Library'" AfterTargets="Build">
```

Comment:
>Condition [](http://example.com/codeflow?start=38&length=9)

only for c++?

Reply:
@huichen123 Will be needed for .NET Framework too, right?

Reply:
i think only .net core 5.0+ is supported for WindowsAppSDK.

Reply:
> only for c++?

Nope. ALL need the Bootstrap.dll. C# _also_ needs Bootstrap.Net.dll (that's just a C# assembly wrapper with P/Invoke over Bootstrap.dll)

Reply:
> @huichen123 Hui Chen FTE Will be needed for .NET Framework (4.x) too, right?

@andrewleader got link to WinAppSDK's prerequisites/supported tech?

Reply:
would C# be handled automatically? or should we simply not copy for AnyCPU?

Reply:
With the folder structure change, you no longer need the targets file for C#/.NET projects as C# msbulid and nuget will do the right thing. Now we can just **delete** the MS.WindowsAppSDK.Bootstrap.targets file and then remove the "import project" from **Build\microsoft.windowsappsdk.foundation.targets** 

C++ projects need another change to **Build\native\windowsappsdk-nuget-native.c.props**. Fixup the sub directory for the native bootstrap dll based on the change above, eg:

- <Command>xcopy /y "$(MSBuildThisFileDirectory)..\..\\**runtimes\win10-$(_WindowsAppSDKFoundationPlatform)\native**\\Microsoft.WindowsAppRuntime.Bootstrap.dll" "$(OutDir)"</Command>

There appears to be an issue with **Build\microsoft.interactiveexperiences.common.targets**. Need to discuss options as it is assuming something if the runtimes/win10-{arch}/native folders exist